source/Projectile.h

```diff
@@ -16,6 +16,7 @@ PARTICULAR PURPOSE.  See the GNU General Public License for more details.
 #include "Body.h"
 
 #include "Angle.h"
+#include "Distribution.h"
```

Comment:
Doesn't belong in the header file, because it isn't used in this file.

Reply:
Yep, that's another one I left behind.  Thanks.

Reply:
Fixed in 2e3ad74c677a45e308fa0105a57179a0bd4e0709.